examples/tree_1d_dgsem/elixir_burgers_shock.jl

```diff
@@ -44,7 +44,7 @@ end
 # Specify non-periodic boundary conditions
 
 function inflow(x, t, equations::InviscidBurgersEquation1D)
-    return initial_condition_shock(coordinate_min, t, equations)
+    return initial_condition_shock(0.0, t, equations)
 end
 boundary_condition_inflow = BoundaryConditionDirichlet(inflow)
```

Comment:
Can't we simplify this even further like

```julia
boundary_condition_inflow = BoundaryConditionDirichlet(initial_condition_shock)
```

?

Reply:
My original version is like yours but I don't know - maybe the author has her/his own ideas about keeping it as a nested function (like many other places here are really tedious for me) you should ask that person.

Reply:
I don't see any benefit of using the more complex version. If we change it right now, we can simplify it. We could look at the git blame and ping people or wait a bit to see whether somebody has to say more about it.

Reply:
You help me ping or we wait together (how long?)

Reply:
Git blame reveals it was #1250 @DanielDoehring. I like `boundary_condition_inflow = BoundaryConditionDirichlet(initial_condition_shock)` because it is concise and independent of the concrete value of `coordinate_min` (0.0 in this case).